Makefile

```diff
@@ -1,22 +1,21 @@
 .PHONY: dg doc build run test clean
 
+TOOLS_IMG = modularitycontainers/image-build-tools
+TOOLS_IMG_RM = true
 DISTRO = fedora-26-x86_64
 VERSION = 2.4
 VARIANT = upstream
-DG = /usr/bin/dg
-GOMD2MAN = /usr/bin/go-md2man
 DOCKERFILE_SRC := Dockerfile.template
 DOCKERFILE := Dockerfile
 TEST_IMAGE_NAME := container-images-tests
 
+IMG_RUN_LINE = docker run --rm=${TOOLS_IMG_RM} -v /var/run/docker.sock:/var/run/docker.sock:Z -v ${PWD}:/src:Z ${TOOLS_IMG}
```

Comment:
Seems like this approach expects that user is able to run containers, which means some non-default configuration of docker ; and a warning is that some people will refuse to configure it (docker community says that if you allow yourself to run containers => you are (almost) privileged as a root)

Reply:
Thanks for the comment. I added a commit that ammends README.md and adds some useful information about this approach. Hopefully that should be enough for now.